contrib/cf/pcf-tile/README.md

```diff
@@ -0,0 +1,26 @@
+# Generate a PCF Tile
+
+1. Install the tile-generator python package. Note: tile-generator requires Python 2, and will not work with Python 3. We recommend using a virtualenv environment to avoid conflicts with other Python packages:
+
+  ```
+  virtualenv -p python2 tile-generator
+  source tile-generator/bin/activate
+  pip install tile-generator
+  ```
+
+  This will put the tile and pcf commands in your PATH.
+
+1. Install the [BOSH CLI v2](https://bosh.io/docs/cli-v2.html)
+
+1. Under the folder `pcf-tile`, execute below command.
+
+  ```
+  ./generate-tile.sh
+  ```
+
+  Note:
+    * Build a major version: ./generate-tile.sh -major
+    * Build a minor version: ./generate-tile.sh -minor
+    * Build a patch version: ./generate-tile.sh
+
+1. You can find open-service-broker-azure-`VERSION`.pivotal under the folder `product`.
```

Comment:
Disclaimer: I'm not a CF guy, really, but I feel like these directions are, possibly, more complicated than they need to be. I have a few thoughts about how to streamline the UX for generating a tile.

1. Is there an existing Docker image that contains the tooling required (e.g. python2, tile-generator, and bosh)? If not, could you consider creating one as its own open source project and publishing that image. It seems it would be generally useful.
1. If the above is addressed, could you consider adding a new make target that will mount project source into a container that contains all the tooling and running the tile-generating script _within_ that container?

For a user, this could streamline the UX to something as simple as:

```
$ make generate-pcf-tile
```

This makes an (fair, I think) assumption that Docker is a more ubiquitous tool than the virtualenv, pythons2, tile-generator, BOSH chain.

Reply:
@krancour OK. I'll try to make it work as what you suggest!